src/server/pause.js

```diff
@@ -0,0 +1,42 @@
+window.d2lTestPause = new Promise(r => window.d2lTestStart = r);
+
+const controls = `
+	<div style="padding: 1rem; background: #ddd; border-bottom: 1px solid #888; color: #222">
+		<div id="start">
+			<button id="start-button" style="margin-inline-end: 1rem;">Start</button>
+			<span>.${window.__WTR_CONFIG__.testFile.split('?')[0]}</span>
+		</div>
+		<div id="run" hidden>
+			<button id="run-button" style="margin-inline-end: 1rem;">Run</button>
+			<span id="test-name"></span>
+		</div>
+	</div>
+`;
+
+document.documentElement.insertAdjacentHTML('afterBegin', controls);
+
+document.querySelector('#start-button').addEventListener('click', start);
+
+const runBtn = document.querySelector('#run-button');
+runBtn.addEventListener('click', run);
+
+const testName = document.querySelector('#test-name');
+
+beforeEach(function() { // eslint-disable-line no-undef
+	testName.innerText = this.currentTest.fullTitle(); // eslint-disable-line no-invalid-this
+	runBtn.disabled = false;
+});
+
+function run() {
+	runBtn.disabled = true;
+	window.d2lTestRun();
+}
+
+function start() {
+	document.querySelector('#start').hidden = true;
+	document.querySelector('#run').hidden = false;
```

Comment:
The focus management here is a bit weird, but maybe there isn't much we can do. I'd expect to want to hit "start", move focus to "run" and then keep it there and hit ENTER a bunch of times until it gets to the test I want to see.

Reply:
Yeah the tests will move focus all over,  but we can always move it back to `run` before we pause.